### PR TITLE
Modifying the Makefile to work both under OS X and Ubuntu

### PR DESCRIPTION
**Pull request by akitaonrails**
It's a rough version, it can be optimised. I did the same PR for crystalized_ruby. Let me know if it works for you.

### CODE REVIEW

**Comment by gaar4ica** [comment]
@akitaonrails Hi Fabio.

There is a problem with this realization for Ubuntu. If I add `gem 'ruby_ext_in_crystal_math', github: 'akitaonrails/ruby_ext_in_crystal_math'` to `Gemfile` it fails to install on `bundle install` with error:

```
make: *** No rule to make target `install'.  Stop.

make install failed, exit code 2
```

Because there is no instruction `install` in Makefile. It came from `mkmf` library that is used for gems with extensions. 

I'm experimenting with it, but haven't succeed yet.

PS: OSX works fine

**Comment by gaar4ica** [comment]
@akitaonrails 

Also I would add this check to `extconf.rb`: 
`abort 'missing llvm-config' unless find_executable('llvm-config')`

**Comment by akitaonrails** [comment]
Hey @gaar4ica just did the fixes, let me know if it works now.

File: ext/common_math/Makefile [hunk 1]
```diff
@@ -14,6 +14,8 @@ LIBS = -levent -lpcre -lgc -lpthread -lruby -ldl -lm -lc
 LDFLAGS = -L. -fstack-protector -rdynamic -Wl,-export-dynamic,-undefined,dynamic_lookup
 TARGET = $(SO_NAME).so
 
+install: $(TARGET)
```
Comment (gaar4ica) [inline ext/common_math/Makefile:17]: That simple 😃 shame on me

**Comment by gaar4ica** [comment]
@akitaonrails works fine. Thank you

**Pull request merged**